pkg/rdt/config.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rdt
+
+import (
+	"fmt"
+)
+
```

Comment:
I think a brief/concise comment section about the cache and memory bandwidth control in RDT vs. the configuration data here, and perhaps a pointer to more detailed documentation about RDT or resctrl would be warranted here.

Reply:
Agree. This isn't yet addressed. Belongs under `documentation` in my todo list :smile: